phpcs.xml.dist

```diff
@@ -8,5 +8,7 @@
         <exclude name="Generic.Files.LineLength"/>
         <exclude name="PEAR.Commenting.ClassComment"/>
         <exclude name="PEAR.Commenting.FileComment"/>
+        <exclude name="Squiz.PHP.CommentedOutCode.Found"/>
```

Comment:
Please add `// phpcs:disable Squiz.PHP.CommentedOutCode` at the top of that one file

Reply:
This would ignore that error in all files.